frodo-runtime/src/test/java/com/fernandocejas/frodo/aspect/LogSubscriberTest.java

```diff
@@ -89,6 +89,19 @@ public void printOnNextMessageBeforeSubscriberOnNextExecution() {
         eq("value"), anyString());
   }
 
+  @Test public void printOnNextMessageBeforeSubscriberOnNextExecutionWithEmptyValues() {
```

Comment:
Test should be written in MessageManager class by passing a null value an asserting its message.